docs/sites.yml

```diff
@@ -4062,3 +4062,15 @@
   built_by: Gaël Billon
   built_by_url: "https://gaelbillon.com"
   featured: false
+- title: Packrs
+  url: "https://www.packrs.co/"
+  main_url: "https://www.packrs.co/"
+  description: >
+    Packrs is a local delivery platform, one spot for all your daily requirements. On a single tap get everything you need at your doorstep.
+  categories:
+    - React Native
```

Comment:
Let's keep these as categories of the _site_ not the app! Otherwise this is fairly confusing :)

Reply:
```suggestion
```

Reply:
@DSchau the website is a landing page for an app build with react native. So, category is suited right?

Reply:
The categories should be representative of the app built with Gatsby, not the underlying product. Does that make sense?

Reply:
understood @DSchau thanks!